TP2/XOR/Xor.tst

```diff
@@ -1,13 +1,9 @@
-// This file is part of www.nand2tetris.org
-// and the book "The Elements of Computing Systems"
-// by Nisan and Schocken, MIT Press.
-// File name: projects/01/Xor.tst
-
+// N'enlevez pas ces lignes
 load Xor.hdl,
 output-file Xor.out,
-compare-to Xor.cmp,

```

Comment:
Ne retire pas cette ligne `compare-to Xor.cmp,`. En local elle est importante pour tester le circuit